gen/schema_gen.go

```diff
@@ -325,7 +325,7 @@ func (g *schemaGen) generate2(name string, schema *jsonschema.Schema) (ret *ir.T
 			}
 			if t.Validators.Float.Set() {
 				switch t.Primitive {
-				case ir.String, ir.ByteSlice:
+				case ir.Float32, ir.Float64, ir.ByteSlice:
```

Comment:
```suggestion
				case ir.Float32, ir.Float64:
```

Reply:
Fixed in https://github.com/ogen-go/ogen/pull/735/commits/dc1c5ea0bda977e6001a117e5bfb7e3b72e09d2cf.